scripts/dev/templates/Dockerfile.e2e

```diff
@@ -3,3 +3,12 @@
 {% block build_binary -%}
 COPY testdata/tls/ testdata/tls/
 {% endblock -%}
+
+
+{% block install_helm -%}
+RUN curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl" \
+    && chmod +x ./kubectl \
+    && mv ./kubectl /usr/local/bin/kubectl \
+    && curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 \
+    && chmod +x get_helm.sh && ./get_helm.sh
+{% endblock -%}
```

Comment:
I just trust you on this :D 

minor nit: can you put `./get_helm.sh` on a separate line as we have each command on a separate line apart from the last 2?